packages/perseus-core/src/widgets/numeric-input/numeric-input-upgrade.ts

```diff
@@ -0,0 +1,37 @@
+import {deriveAnswerForms} from "./util";
+
+import type {PerseusNumericInputWidgetOptions} from "../../data-schema";
+
+export const currentVersion = {major: 1, minor: 0};
+
+export const widgetOptionsUpgrades = {
```

Comment:
Upgrader using the `widgetOptionsUpgrades` API (which will likely be replaced by Ben's parser)